Mathlib/Data/NNRat/Lemmas.lean

```diff
@@ -0,0 +1,95 @@
+/-
+Copyright (c) 2022 Yaël Dillies, Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Bhavik Mehta
+-/
+import Mathlib.Algebra.Function.Indicator
+import Mathlib.Algebra.Order.Nonneg.Field
+import Mathlib.Data.NNRat.Defs
+
+#align_import data.rat.nnrat from "leanprover-community/mathlib"@"b3f4f007a962e3787aa0f3b5c7942a1317f7d88e"
+
+/-!
+# Algebraic structures on the nonnegative rationals
+
+This file provides additional results about `nnrat` that cannot live in earlier files due to import
+cycles.
+-/
+
+open Function
+open scoped NNRat
+
+deriving instance CanonicallyLinearOrderedSemifield, LinearOrderedCommGroupWithZero for NNRat
```

Comment:
Doesn't the first instance imply the second

Reply:
Yes, but this is a shortcut instance.